docs/reference/resource-profiling.md

```diff
@@ -39,83 +39,95 @@ The resource tests were intended to address the following problem statements:
 
 ## Baseline Resource Requirements
 
-This section captures the results of tests to determine minimum resource requirements for the K3s agent, the K3s server with a workload, and the K3s server with one agent.
+This section captures the results of tests to determine minimum resource requirements for basic K3s operation.
 
 ### K3s Server with a Workload
 
-These are the requirements for a single-node cluster in which the K3s server shares resources with a workload.
+These are the requirements for a single-node cluster in which the K3s server shares resources with a [simple workload](https://kubernetes.io/docs/tasks/run-application/run-stateless-application-deployment/).
 
 The CPU requirements are:
 
-| Resource Requirement | Tested Processor  |
-|-----------|-----------------|
-| 10% of a core | Intel(R) Xeon(R) Platinum 8124M CPU, 3.00 GHz |
-| 20% of a core | Low-power processor such as Pi4B BCM2711, 1.50 GHz |
+| System | CPU Core Usage |
+|--------|----------------|
+| c6id.xlarge | 6% of a core |
+| Pi4B | 30% of a core |
+
+The Memory Requirements are:
+
+| Tested Datastore | System | Memory |
+|-----------|---------|------|
+| Kine/SQLite | c6id.xlarge | 1602 M | 
+|             | Pi4B |  1588 M |
+| Embedded etcd | c6id.xlarge | 1604 M | 
+|               | Pi4B |  1613 M |
 
-The IOPS and memory requirements are:
+The Disk requirements are:
 
-| Tested Datastore | IOPS | KiB/sec | Latency | RAM  |
-|-----------|------|---------|---------|--------|
-| Kine/SQLite | 10 | 500     | < 10 ms | 768 M  |
-| Embedded etcd | 50 |  250  | < 5 ms  | 896 M  |
+| Tested Datastore | IOPS | KiB/sec | Latency |
+|-----------|------|---------|---------|
+| Kine/SQLite | 10 | 500     | < 10 ms |
+| Embedded etcd | 50 |  250  | < 5 ms  |
 
 ### K3s Cluster with a Single Agent
 
 These are the baseline requirements for a K3s cluster with a K3s server node and a K3s agent, but no workload.
 
+#### K3s Server
 The CPU requirements are:
 
-| Resource Requirement | Tested Processor  |
-|-----------|-----------------|
-| 10% of a core | Intel(R) Xeon(R) Platinum 8124M CPU, 3.00 GHz |
-| 20% of a core | Pi4B BCM2711, 1.50 GHz |
+| System | CPU Core Usage | 
+|--------|----------------|
+| c6id.xlarge | 5% of a core |
+| Pi4B | 25% of a core |
 
-The IOPS and memory requirements are:
+The Memory Requirements are:
 
-| Datastore | IOPS | KiB/sec | Latency | RAM    |
-|-----------|------|---------|---------|--------|
-| Kine/SQLite | 10 | 500     | < 10 ms | 512 M  |
-| Embedded etcd | 50 |  250  | < 5 ms  | 768 M  |
+| Tested Datastore | System | Memory |
+|-----------|---------|------|
+| Kine/SQLite | c6id.xlarge | 1428 M |
+|             | Pi4B |  1215 M |
+| Embedded etcd | c6id.xlarge | 1450 M |
+|               | Pi4B |  1413 M |
 
 ### K3s Agent
 
-The CPU requirements are:
+The requirements are:
+
+| System | CPU Core Usage | RAM |
+|--------|----------------|-----|
+| c6id.xlarge | 3% of a core | 275 M |
```

Comment:
Can we keep this as a CPU model/speed? I have no idea what a c6id instance uses for the processor.